test/browser_tests/spec_suites/homepage.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+function _getStylesheets() {
+  var sheets = document.styleSheets;
+  var hrefs = [];
+
+  for ( var i = 0; i < sheets.length; i++ ) {
+    hrefs.push( sheets[i].href );
+  }
+
+  return hrefs;
+}
+
+describe( 'The homepage', function() {
+
+  beforeEach( function() {
+    browser.get( '/' );
+  } );
+
+  it( 'should load the stylesheet', function() {
+    browser.executeScript( _getStylesheets ).then(
+      function( styleSheets ) {
+        browser.styleSheets = styleSheets;
+
+        browser.getCapabilities().then( function( cap ) {
+          browser.name = cap.get( 'browserName' );
```

Comment:
I'm not sure if this is necessary due to how `Control Flow ordering`. https://github.com/seleniumhq/selenium/issues/677. I just don't know enough about it and it's within Protractor.